INSTALL.md

```diff
@@ -479,6 +482,34 @@ default.debug=true
 EOL
 ```
 
+## 3.12. Building on Arch Linux
+
+### 3.12.1. Install Build Dependencies
+
+```bash
+sudo pacman -S pyqt-builder gsl gpsbabel postgresql sip libzip qtkeychain-qt5 qca-qt5 qscintilla-qt5 expat qwt gdal \
+spatialindex sqlite geos proj qt5 libspatialite cmake gcc flex bison
+```
+
+### 3.12.2. Compiling
+
+In the root of the repository, at any tag or branch you choose:
+
+Keep in mind, this will destroy any contents of your build directory if you have one.
+
+```bash
+if [ -d ./build/ ]; then rm -rfv ./build/*; else mkdir -v build; fi
+cd build
+cmake ..
+make
+```
+
+Optionally:
+
+```bash
+sudo make install
```

Comment:
This should have some context.
There are other methods to run a self-compiled qgis (directly from the build dir or installing into a user directory) which I prefer in almost any case. Those who are at ease with system administration will know that this is an option even without this documentation. Those who need this documentation (and will just copy paste it) are probably not aware that it's not that easy to undo.